gossip/src/cluster_info.rs

```diff
@@ -392,8 +384,7 @@ fn retain_staked(values: &mut Vec<CrdsValue>, stakes: &HashMap<Pubkey, u64>) {
             // the various dashboards.
             CrdsData::Version(_) => true,
             CrdsData::NodeInstance(_) => true,
-            // getHealth fails if account hashes are not propagated.
-            CrdsData::AccountsHashes(_) => true,
+            CrdsData::AccountsHashes(_) => false,
```

Comment:
The consequence of this change is that:
* your node sends a pull request to another node asking for missing values.
* that node responds with account hashes that your node does not have.
* next time your node sends a pull request, that node will respond again with those same account hashes.

So it will exacerbate gossip traffic with repetitive pull responses which are not ingested.

Reply:
question on this. wouldn't `gossip/src/cluster_info.rs#2105-#2109` prevent the pull response from sending back these `AccountHashes` if we have: `CrdsData::AccountsHashes(_) => false`

Reply:
responded here: https://github.com/solana-labs/solana/pull/34979#discussion_r1470282417